packages/@aws-cdk/aws-appmesh/lib/virtual-node.ts

```diff
@@ -89,7 +528,7 @@ export interface VirtualNodeBaseProps {
    *
    * @default - No listeners
    */
-  readonly listener?: VirtualNodeListener;
+  readonly listeners?: VirtualNodeListener[];
```

Comment:
Why this change....? Considering there can only be a single Listener according to the validation above, I don't think this change is needed.

Reply:
As discussed offline with @dfezzie, we will be accepting the array of listeners for Virtual Node without validation.

Reply:
Then you need to add a BREAKING CHANGE related to this. See: https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md#step-4-commit

Reply:
Still missing the breaking change in the PR description.